### PR TITLE
Remove readmes and point to docs

### PR DESCRIPTION
**Pull request by moxiegirl**
First part of a two part process.  
Remove the readmes and point to docs.docker.com.
Anything that is about contributing, I'll leave in the readme.

Signed-off-by: Mary Anthony <mary@docker.com>

### CODE REVIEW

**Comment by moxiegirl** [comment]
@abronan didn't move `leadership/README.md` i hear from @dongluochen that you are working on this. We can update later.  The actual readme hasn't been touched in 5 months.  Does this work for you?

**Comment by abronan** [comment]
@moxiegirl yes, sounds good.

LGTM

@vieux @dongluochen PTAL

File: cluster/mesos/README.md [hunk 1]
```diff
@@ -1,73 +1,10 @@
...
-If you run into `Abnormal executor termination` error, you might want to run the
-swarm container with an additional environment variable:
-`SWARM_MESOS_USER=root`.
+schedule container resources.. To read the end-user documentation, visit the
+[Swarm scheduler documentation on
+docs.docker.com](https://docs.docker.com/swarm/scheduler/strategy/). If you want
+to modify the scheduler end-user documentation, start with [the
+`/docs/scheduler`
+files](https://github.com/docker/swarm/tree/master/docs/scheduler) in this
+project.
```
Comment (dongluochen) [inline cluster/mesos/README.md]: There is no corresponding file under docs/ for cluster/mesos/README.md. We may want to keep this file.

File: discovery/README.md [hunk 2]
```diff
@@ -1,253 +1,6 @@
...
-And for the `nodes` discovery method.
-
-```bash
-swarm manage -H <swarm_ip:swarm_port> "nodes://10.0.0.[10:200]:2375,10.0.1.[2:250]:2375"
-```
-
-## Contributing a new discovery backend
+Docker Swarm comes with multiple Discovery backends. To read the end-user documentation, visit the [Swarm discovery documentation on docs.docker.com](https://docs.docker.com/swarm/discovery/). If you want to modify the discovery end-user documentation, start with [the `docs/discovery.md` file](https://github.com/docker/swarm/blob/master/docs/discovery.md) in this project.
 
 Contributing a new discovery backend is easy, simply implement this
```
Comment (dongluochen) [inline discovery/README.md]: This line and below should be removed.

File: discovery/token/README.md [hunk 3]
```diff
@@ -1,29 +1,9 @@
...
-
-#####Delete a cluster (all the nodes in a cluster)
-`-> DELETE https://discovery.hub.docker.com/v1/clusters/<token>`
-
-`<- OK`
+Docker Swarm comes with a simple discovery service built into the [Docker
+Hub](http://hub.docker.com).  To read the end-user API documentation, visit [the
+Swarm API documentation on
+docs.docker.com](https://docs.docker.com/swarm/api/swarm-api/). If you want to
+modify the discovery API documentation, start with [the `docs/api/swarm-api.md`
```
Comment (dongluochen) [inline discovery/token/README.md]: discovery/token/README.md is covered by docs/discovery.md

File: cluster/mesos/README.md (hunk 1 above)
Comment (moxiegirl) [inline cluster/mesos/README.md]: @dongluochen I am thinking we should add this as a full blown example in 1.10.  WDYT?

File: discovery/token/README.md (hunk 3 above)
Comment (moxiegirl) [inline discovery/token/README.md]: Ok, so remove the entire file?

File: Godeps/Readme [hunk 4]
```diff
@@ -1,3 +1,5 @@
+# DO NOT MODIFY
```
Comment (vieux) [inline Godeps/Readme]: This is automatically generated, we shouldn't touch this file.

File: cluster/mesos/README.md (hunk 1 above)
Comment (vieux) [inline cluster/mesos/README.md]: @moxiegirl I'm not sure about that, for now it's still experimental, so I would leave this readme here.

File: discovery/token/README.md (hunk 3 above)
Comment (dongluochen) [inline discovery/token/README.md]: @moxiegirl I think we can keep a simple README.md in this folder. Something like this: 

```
Docker Swarm comes with a simple discovery service built into the [Docker
Hub](http://hub.docker.com). To read the end-user API documentation, visit [the
Swarm discovery documentation on
docs.docker.com](https://docs.docker.com/swarm/discovery/). If you want to
modify the discovery documentation, start with [the `docs/discovery.md`]
```

File: cluster/mesos/README.md (hunk 1 above)
Comment (moxiegirl) [inline cluster/mesos/README.md]: OK @vieux .  Are we ok with using the word experimental in the file?

File: Godeps/Readme (hunk 4 above)
Comment (moxiegirl) [inline Godeps/Readme]: _dope slaps herself_

got it.  thanks for the catch @vieux

**Comment by moxiegirl** [comment]
@dongluochen Sorry for the delay.  One last review please.

**Comment by vieux** [comment]
@moxiegirl could you please update `Godeps/Readme`  so there is no change in it (last line) ? thanks

**Comment by moxiegirl** [comment]
@abronan done!

**Pull request merged**